scio-bigquery/src/test/scala/com/spotify/scio/bigquery/types/SchemaProviderTest.scala

```diff
@@ -77,4 +77,32 @@ class SchemaProviderTest extends FlatSpec with Matchers {
     SchemaProvider.schemaOf[RepeatedNested] should equal (parseSchema(recordFields("REPEATED")))
   }
 
+  case class User(@description("user name") name: String, @description("user age") age: Int)
+  case class Account(@description("account user") user: User,
+                     @description("in USD") balance: Double)
```

Comment:
I guess it's fine - but another option to test could be mixed - class with and without description fields.